src/interpreter.rs

```diff
@@ -545,7 +756,28 @@ pub fn eval_script(
                                 return Err(ScriptError::InvalidStackOperation);
                             }
 
-                            todo!()
+                            if let Ok(vch2) = stack.pop() {
```

Comment:
These `if let`s are redundant with respect to the stack size check; they could just be `let ... = stack.pop()?;` instead.